src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1439,4 +1439,32 @@ public static void setFilteredGenotypeToNocall(final VariantContextBuilder build
         // update genotypes
         builder.genotypes(genotypes);
     }
+
+    /**
+     * Calculate the Genotype Quality (GQ) by subtracting the second smallest Phred-scaled likelihoods (PL) first smallest.
```

Comment:
Thanks for adding javadoc!  But "subtracting the smallest Phred-scaled likelihood (PL) from the second smallest" is more accurate.